mltrace/cli/cli.py

```diff
@@ -274,16 +294,27 @@ def flag(output_id: str, address: str = ""):
 
 
 @mltrace.command("unflag")
-@click.argument("output_id")
+@click.option("--output_id", help="Output ID to unflag")
+@click.option("--all", is_flag=True)
 @click.option("--address", help="Database server address")
-def unflag(output_id: str, address: str = ""):
+def unflag(output_id: str = "", all: bool = False, address: str = ""):
     """
-    Command to set the flag property of an output_id to false.
+    Command to set flag property of an output_id or all output_ids to false.
     """
     # Set address
     if address and len(address) > 0:
         set_address(address)
-    unflag_output_id(output_id)
+
+    if all:
+        outputs, component_counts = review_flagged_outputs()
```

Comment:
Write a `store.py` and `client.py` function to directly get all the flagged iops and delete them:

```
flagged_iops = (
            self.session.query(IOPointer)
            .filter(IOPointer.flag.is_(True))
            .all()
        )

for iop in flagged_iops:
  iop.clear_flag()

self.session.commit()
```

Reply:
done